monai/bundle/scripts.py

```diff
@@ -231,10 +236,16 @@ def _download_from_ngc(
 
 
 def _download_from_ngc_private(
-    download_path: Path, filename: str, version: str, remove_prefix: str | None, repo: str, headers: dict | None = None
+    download_path: Path,
+    filename: str,
+    version: str,
+    repo: str,
+    prefix: str = "monai_",
+    remove_prefix: str | None = "monai_",
```

Comment:
Look like the change of the default value here will make the behavior not consistent with previous.
If no special reason, I would suggest revert it back.
cc @mingxin-zheng

Reply:
no, the default is always monai_. Please check: https://github.com/Project-MONAI/MONAI/blob/23ade147fd9a3c32d754b28c2ae4c365b924b04a/monai/bundle/scripts.py#L440

Reply:
My bad, closed as resolved. Thanks.